### DISCUSSION TITLE
SR-OCC-2025-801: Saving Pensions

### DISCUSSION THREAD

**Discussion by JFWooten4** [WhyDRS/SEC-comments#14] [Comment Letters]
As we’ve discussed for over [a year](https://lnns.co/0clzI2NVtmk) now, Block Transfer purports to offer the first transfer agent depository.[^me] They use a public blockchain called Stellar to track shares directly between companies and investors like us. Since this is the first scalable, fungible, and automated application of DRS, we have, for the first time, a system that can truly replace the outdated and potentially insolvent CCP regime presently in place.

[^me]: I am writing this Discussion and subsequent comment on my own accord and not necessarily on behalf of or through employment with BlockTrans Syndicate. I will make this distinction more explicit in the final comment, so as to act as a point of reference for future WhyDRS regulatory perspectives. This gives me a unique opportunity to objectively analyze the position of the Syndicate in the context of legacy TA regulations.

## Background and Context

If any of the words or concepts in that paragraph didn’t make sense to you, I highly recommend reaching out to the community for a chat. I am happy to answer any Stellar questions in the `#tad3-and-blocktransfer` channel, and there are so many incredible community members to help with DRSing questions in general. As for the TAD concept itself, there was a good [ape discussion](https://www.reddit.com/r/UrvinFinance/comments/ttw9st/t0_is_not_enough_tell_the_sec_that_you_want_full/) on this which complements the [primary sources](https://www.blocktransfer.com/blog/post/introducing-transfer-agent-depository) _per relationem_, especially from Dr. Trimbath.

[This proposed rule](https://www.federalregister.gov/documents/2015/12/31/2015-32755/transfer-agent-regulations) is one of the few SEC documents discussing the TAD concept post-1980, and I learned quite a few things on my first read-through a couple of years ago. One image, in particular, well lays out the rampant complexities of the central brokers, and this was released all the way back in 2015:

[<img width="640" alt="SEC" src="https://github.com/user-attachments/assets/ddf2995c-79a6-4af9-b566-c153c138ad8d">](https://www.govinfo.gov/content/pkg/FR-2015-12-31/pdf/2015-32755.pdf#page=12)

For me, the exciting part is that we basically get to rebuild the system anew with efficiency at its core since no other TAs have built a TAD before.

On that point, I think it’s relevant for me to disclose that I don’t see a future where we get all the benefits of true ownership and none of the risks of falling back into today’s problems—unless we commit to Stellar. I have known this in my heart for at least five years, and I have done everything I can in the last couple to think of any alternative. But after extensive contemplation, there does not seem to be any other viable present alternative, and I’d be happy to dive into the logistical proofs for anything proposed.[^comm]

[^comm]: Namely, I’ve been asking the community since joining if they could think of any other better system. Aside from explicitly centralized systems which we’ve debunked thoroughly, there does not seem to exist any other sufficiently decentralized exchange or blockchain capable of becoming the web3 stock market. Again, if you want to get filled in on these conversations or chat about something you’re passionate about, I’m happy to engage here or in Discord—as I’m sure many other community members would appreciate.

## Chief Ambition

That said, there is something else much more sinister and pressing that I believe should be the sole focal aim of this comment:

[<img width="800" alt="UCC" src="https://github.com/user-attachments/assets/617e2356-3e5f-4dc8-95fe-cbe4da4354a3">](https://thegreattaking.com/)

As I understand it, there are at least 8 U.S. States which are presently amending their Uniform Commercial Codes to prevent the theft of our investments by Wall Street. In at least one state, banking lobbyists have threatened to remove all financial service operations should a local change happen.

Once the first few states start undoing the lies they pushed through regional laws in 1994, there could be an exodus of assets into these areas of greater protection. If large institutions choose to take advantage of these increased securities by moving the geography of their custodians, then the present CCP regime could face material difficulties in solvently operating. We’ve discussed these consequences quite a bit in the DUNA meetings and through past publications (including from myself) in the last year, and I’d be happy to keep the conversation going across the industry.

That said, there is a panacea answer of sorts that can seamlessly stabilize this treacherous situation, protecting investors, maintaining markets, and encouraging confidence. Americans presently residing in a select few political jurisdictions with influence over the Federal government can pass rules protecting all investors, not just those in states adopting the simple change to U.C.C. Art. 8. This is called “preemption” of regional laws by the authoritative law of the land.

## Next Steps

It is my understanding that the present administration made a promise to keep us fortified from Wall Street’s looming evisceration:

[<img width="444" alt="RFK" src="https://github.com/user-attachments/assets/ea572c05-6457-433c-b2af-62f1c06333dd">](https://x.com/RobertKennedyJr/status/1792970117204287992)

If the CCP regime fails before all U.S. States amend their U.C.C. Articles, hundreds of millions of Americans stand to lose their savings and retirements due to the problematic waterfall structure we challenged successfully this year in the OCC and NSCC petitions. Thankfully, a Federal defense will insulate us from these unwarranted, unknown, and unfair market risks.

Now is our chance to make our voices heard in the turmoil of a Commission transition towards distributed ledger technology. In the past, we have invoked exceptional attention and change through large amounts of individual outreach through a central entity. The name of a single reference holds its weight in gold from the perspective of SEC staff.[^infs]

[^infs]: I make this claim based on years of reading thousands of comments on hundreds of rules. It is quite clear both historically and from personal experience that collective comment letters receive much more serious contemplation by staff. While they may personally acknowledge an individual comment, the vast majority of citations in financial rules reference collective organizations, and this is the chief method of understanding your impact on government policy.

But it doesn’t need to all come from one person—indeed it likely should not. We have a particularly unique situation with this rule, because it has been:

- Out for comment for nearly a decade,
- On the SEC’s [regulatory agenda](https://www.sec.gov/rules-regulations/2024/10/regulatory-agenda-semiannual-regulatory-agenda) for years, and
- Received no comments in over three years.

Accordingly, a single, collaborative, well-executed letter from the DUNA could be the targeted shot we’ve been waiting for to get back into conversations with Commissioners.

Namely, the SEC cannot adopt new rules based on S7-27-15 since it’s so old. They have specific processes to follow which graciously require thorough analysis of public opinion. Therefore, we have a rare opportunity to write an emotionally compelling letter which (i) gets the Federal preemption point across and (ii) requests staff to update their perspective on transfer agent regulations.

I’ve contacted staff asking for clarity on the TARs at least three times this year, with one instance referencing the regulatory agent which states TARs are of chief importance to the Commission. Their Spring declaration also got a lot of Dr. T plumbing comments supporting the update of TARs. In all instances, they have (respectfully) stonewalled me to the overarching Federal government administration references, which have not proved fruitful through my own efforts.

To put it simply, all we need is a rule saying banks, brokers, and custodians don’t get to steal your portfolio to pay off their debts. Now is our moment to share our DRS passions and ownership mission with the new Commission so that Main Street doesn’t get stuck holding the bag for these unadulterated thieves. So, what thoughts do you have that we can incorporate into [this comment letter](https://github.com/WhyDRS/SEC-Comments/blob/main/TAR-S7-27-15.md) about TARs?

**Comment by JamesAlfonse** [comment]
Fantastic read, John. Appreciate all the time you put into every single aspect shown here and in the comment letter, including: creating the blueprint to an open-source transfer agent on a decentralized blockchain, researching laws and examples to make your points and relating them with personal anecdotes, reaching out (multiple times) to the SEC as well as transfer agents, offering to gift your shares to the Association and requesting a response from the SEC on this action, and overall putting everything together and getting things in motion. Thank you for your efforts, and I hope that as more people join the Association and contribute, they can also understand how important it is to push for these changes.

**Reply by JFWooten4** [reply]
Thank you for looking over it, James! Just these last few days, having some more eyes scan through has been super helpful in realizing how much more work to put into it in preference over getting something out quick. I'm glad those aspects stood out, especially the multiple SEC outreaches. It's challenging because I want to only communicate in public, auditable forums over backroom opaque conversations. Thus, everything is going through a much slower rulemaking review time period. But I've had a few years to get used to the pace, and I'm very impressed by just how much internally they've been doing, through inference from publications like the new CCP closure rules, to protect investors as best they can given the circumstances.

P.S. Saw and appreciate some of the critiques in Discord, which I'll get back to there. For public docs, it will center around bringing some of the key points up first and foremost, which I'm working towards now in #24. 🔐

**Comment by JFWooten4** [comment]
I've repositioned this letter as a response to `SR-OCC-2025-801` for reasons which should be apparent in the final draft. Generally, this shift ought to better align the Chief Ambition with a mode of effectuating changes.

**Comment by admin** [comment]
Comment [submitted](https://www.sec.gov/comments/sr-occ-2025-801/srocc2025801-598095-1737722.pdf). It covered the substantive CCP points, set up the introduction for a TAR which was contemplated in [BlockTransfer's first exam](https://github.com/JFWooten4/agenda/issues/7), and we are now past the stated release of TA-specific proposed rules.

[<img width="947" height="436" alt="www.sec.gov/rules/concept/2015/34-76743.pdf" src="https://github.com/user-attachments/assets/74305f7e-28a2-400b-a8f3-ff124c505b2d" />](https://www.reginfo.gov/public/do/eAgendaViewRule?pubId=202504&RIN=3235-AL55)

Since all the Webb patterns are substantiated, we can reopen a new discussion with new intent whenever they get back around to S7–27–15.
